bears/r/FormatRBear.py

```diff
@@ -78,10 +78,13 @@ def create_arguments(filename, file, config_file,
         """
         options = {'source="' + escape(filename, '"\\') + '"',
                    'blank=' + _map_to_r_bool(r_keep_blank_lines),
-                   'brace.newline=' + _map_to_r_bool(r_braces_on_next_line),
                    'comment=' + _map_to_r_bool(r_keep_comments),
                    'arrow=' + _map_to_r_bool(r_use_arrows),
                    'indent=' + str(indent_size)}
+        if r_braces_on_next_line is not None:
```

Comment:
isnt `if r_braces_on_next_line is not None`` equivalent to ``if r_braces_on_next_line``?